indico/modules/categories/controllers/management.py

```diff
@@ -370,9 +389,13 @@ def _process(self):
 
 
 class RHSortSubcategories(RHManageCategoryBase):
-    def _process(self):
+
+    @use_kwargs({
+        'categories': fields.List(fields.Int())
```

Comment:
`required=True`; otherwise this fails since it won't pass the kwarg if it's missing in the request data